### DISCUSSION TITLE
Manage notes / excerpts / quotes of a book as a listlist

### DISCUSSION THREAD

**Discussion by vz-chameleon** [mateusz-bak/openreads#726] [Ideas]
I really openreads but I think it might be currently lacking a way to add favourite quotes / excerpts of a reading. It is currently possible to add "Notes" to an entry, so selected quotes could be put there... but it's less user-friendly and not readable

**Notes section is conceived as the review section**
It is a single text section, but selected quotes could be added as an additional "list of notes" that can be added as we read, and then displayed on demand when opening a book entry (maybe in carroussel form ?).

In any form, I think it could be interesting to add notes aboht what one finds interesting during reading.

I'd like to develop this, do you think it's possible with Flutter and in regards to the representation/memory storage of the book items ?

Hoping I've managed to explain the feature suggestion clearly, thank you in advance for any help !

**Comment by pandalamp** [comment]
A carousel option sounds neat, but I am not sure this is a practical idea for this app. First, there are already large text sections for Description, My Review, and Notes for you to record whatever you like. Second, as a HUGE highlighter, I know I wouldn't be interested in importing bulk quotes / highlights from especially electronic sources (say Goodreads, Kindle, Libby) one-by-one into a _list of notes_ as you describe. But maybe you would! 😅

Again, I like your idea and your willingness to work on it, but I feel less sure of how useful it would actually be if in the end the best approach would be to add yet another (and unlimited size) text field.
